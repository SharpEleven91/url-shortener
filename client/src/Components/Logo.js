import React from "react";
import { Typography } from "@material-ui/core/";

const Logo = () => {
  return (
    <Typography data-testid="logo" color="inherit" component="h2" variant="h2" gutterBottom>
      ./URL MiNiFy
    </Typography>
  );
}

export default Logo