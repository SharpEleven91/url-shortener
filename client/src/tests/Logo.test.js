import React from 'react';
import expect from 'expect';
import Logo from '../Components/Logo';
import { render, cleanup } from 'react-testing-library';
import "jest-dom/extend-expect";
afterEach(cleanup);
test('Component renders properly', () => {
    const { queryByTestId } = render(<Logo/>);
    expect(queryByTestId("logo")).toHaveTextContent("./URL MiNiFy");
    expect(queryByTestId("logo")).toMatchSnapshot();
});
