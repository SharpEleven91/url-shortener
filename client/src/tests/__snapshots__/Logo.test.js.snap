// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component renders properly 1`] = `
<h2
  class="MuiTypography-root-1 MuiTypography-h2-14 MuiTypography-colorInherit-30 MuiTypography-gutterBottom-28"
  data-testid="logo"
>
  ./URL MiNiFy
</h2>
`;
